test-project/Assets/EditmodeTests/ModularCodegenTests.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Linq;
+using System.Reflection;
+using NUnit.Framework;
+
+namespace Improbable.Gdk.EditmodeTests
+{
+    [TestFixture]
+    public class ModularCodegenTests
+    {
+        private Assembly generatedCodeAssembly;
+
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            // Assembly.FullName == "<name> <version> <culture> <publickeytoken>", so we just .Contains() it.
+            generatedCodeAssembly = AppDomain.CurrentDomain.GetAssemblies()
```

Comment:
It might be safer to use `typeof` on a known type in the Generated assembly, and get the `.Assembly` through that.